File: buymesth-front/src/app/models/OperationStatusForm.ts
import {OperationStatus} from "./OperationStatus";

export class OperationStatusForm {

    idOperation: number;
    idUser: number;
    status: OperationStatus;

    constructor(idOperation: number, idUser: number, status: OperationStatus) {
        this.idOperation = idOperation;
        this.idUser = idUser;
        this.status = status;
    }
}

File: buymesth-back/src/main/java/com/buymesth/app/dtos/OperationStatusForm.java
package com.buymesth.app.dtos;

import io.swagger.annotations.ApiModel;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.validation.constraints.Min;
import javax.validation.constraints.NotBlank;
import javax.validation.constraints.NotNull;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@ApiModel
public class OperationStatusForm {

    @NotNull
    @Min(0)
    private Long idOperation;

    @NotNull
    @Min(0)
    private Long idUser;

    @NotNull
    @NotBlank
    private String status;
}
